### PR TITLE
Automated cherry pick of #15491

### PR DESCRIPTION
**Pull request by bprashanth**
Cherry pick of #15491 on release-1.1.

### CODE REVIEW

**Comment by bprashanth** [comment]
I think CLA is just confused.
@derekwaynecarr @thockin

**Comment by thockin** [comment]
CLA bot will never be OK in this case - it's on humans to verify.

**Comment by thockin** [comment]
Assuming this passes tests, you can manually merge it - merge bot won't because of CLA.

**Comment by nikhiljindal** [comment]
@bprashanth Can you please look at the shippable failure?
This is going to make cherrypicking https://github.com/kubernetes/kubernetes/pull/15516 a lot easier :)

**Comment by bgrant0607** [comment]
LGTM

File: pkg/apis/extensions/validation/validation_test.go [hunk 1]
```diff
@@ -932,3 +932,209 @@ func newInt(val int) *int {
...
+			s := strings.Split(k, ":")
+			err := errs[0].(*errors.ValidationError)
+			if err.Field != s[0] || !strings.Contains(err.Error(), s[1]) {
+				t.Errorf("unexpected error: %v, expected: %s", errs[0], k)
+			}
+		}
+	}
+}
+
+func TestValidateClusterAutoscaler(t *testing.T) {
```
Comment (bgrant0607) [inline pkg/apis/extensions/validation/validation_test.go]: You picked up some unintended changes here. Please revert the ClusterAutoscaler stuff.

**Comment by bgrant0607** [comment]
The Shippable failure is due to stuff in pkg/apis/extensions/validation/validation_test.go that shouldn't be there.

**Comment by wojtek-t** [comment]
@bprashanth - please rebase

**Comment by bprashanth** [comment]
Sorry about the rebase conflict, didn't notice till just now. Waiting on shippable.

**Comment by bprashanth** [comment]
tests passed, merging manually

**Pull request merged**